.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+### Description
+What does this do? What problem/s does it solve? What issue numbers does it
+resolve?
+
+### Checklist
+- [ ] I have read [CONTRIBUTING.md](https://github.com/sopel-irc/sopel/blob/master/CONTRIBUTING.md)
+- [ ] My patch is based on the correct branch (eg. `6.6.x` or `master`)
+- [ ] I am opening this PR against that branch
+- [ ] I can and do license this code under EFLv2
```

Comment:
Nitpick: Contributions aren't necessarily code.

```suggestion
- [ ] I can and do license my contributions under the EFLv2
```

Reply:
Fair. I said "this" instead of "my" in case a contribution includes things not written by the PR-opener.